src/app/api/health/route.ts

```diff
@@ -1,9 +1,11 @@
+"use server";
+
 import packageJson from "../../../../package.json";
 import { NextRequest, NextResponse } from "next/server";
 
 export async function GET(_: NextRequest) {
   const version = packageJson.version;
-  const contracts_deployer = process.env.NEXT_PUBLIC_CONTRACTS_DEPLOYER;
+  const contracts_deployer = process.env.CONTRACTS_DEPLOYER;
```

Comment:
This is missing from the `.env.sample`

Reply:
yes must have missed it!

Reply:
Now using SBTC_CONTRACT_ADDRESS instead of duplicating code